src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -307,6 +324,22 @@ private void RefreshCondensedDisplay(Action refreshDisplay)
 
             string newContent = "null";
 
+            if (mirrorData is MirrorDataInProgress
+                // When CBN is created, its cached value is set to MirrorDataInProgress.
+                // And it's not updated after update graph task is complited.
+                && (nodeViewModel.NodeLogic as CodeBlockNodeModel == null || (nodeViewModel.NodeLogic as CodeBlockNodeModel).Code != String.Empty))
```

Comment:
@ke-yu , @Randy-Ma this check is not useless. It's situation, when CBN is just created. If we won't have this check, we will get the next situation:
![image](https://cloud.githubusercontent.com/assets/8158404/13630026/a2bb3154-e5e5-11e5-8270-0b137501dcf7.png)